packages/neo-one-node-http-rpc/src/middleware/tooBusyCheck.ts

```diff
@@ -0,0 +1,17 @@
+import { Context } from 'koa';
+import toobusy_js from 'toobusy-js';
+
+export interface Options {
+  readonly maxLag?: number;
+}
+
+export const tooBusyCheck = (options: Options) => async (ctx: Context, next: () => Promise<void>) => {
+  if (options !== undefined && options.maxLag !== undefined) {
+    toobusy_js.maxLag(options.maxLag);
```

Comment:
Set this outside of the request handler (i.e. one lambda up). No reason to be setting this on every request.

Is this the only option available?